src/ZenohUTransport.cpp

```diff
@@ -26,23 +26,21 @@ const uint32_t WILDCARD_ENTITY_ID = 0x0000FFFF;
 const uint32_t WILDCARD_ENTITY_VERSION = 0x000000FF;
 const uint32_t WILDCARD_RESOURCE_ID = 0x0000FFFF;
 
-using namespace zenoh;
-using namespace uprotocol::v1;
-using namespace uprotocol::datamodel;
-
-UStatus ZenohUTransport::uError(UCode code, std::string_view message) {
-	UStatus status;
+uprotocol::v1::UStatus ZenohUTransport::uError(uprotocol::v1::UCode code,
+                                               std::string_view message) {
+	uprotocol::v1::UStatus status;
```

Comment:
Minor: since this code is already inside the `uprotocol` namespace, the `uprotocol::` prefix isn't required. For example, this could be:
```suggestion
v1::UStatus ZenohUTransport::uError(v1::UCode code,
                                               std::string_view message) {
	v1::UStatus status;
```